packages/contracts/src/dollar/interfaces/IERC1155Ubiquity.sol

```diff
@@ -32,5 +22,5 @@ interface IERC1155Ubiquity is IERC1155 {
 
     function exists(uint256 id) external view returns (bool);
 
-    function holderTokens() external view returns (uint256[] memory);
+    function holderTokens(address holder) external view returns (uint256[] memory);
```

Comment:
Curious why this was added too.

Reply:
here @kfujita0520  is right. we were missing `address holder` parameter.

Reply:
> > It looks like you updated a lot of interfaces. Is this necessary?
> 
> @pavlovcik Thank you very much for your comments and review. The main logic resides in only one contract: DirectUBQFarmer(DirectGovernanceFarmer). the rest of contract is just interfaces which need to interact with external contract and nothing impact on your existing system. Meanwhile I changed one existing interface
> 
> ```
> -    function holderTokens() external view returns (uint256[] memory);
> +    function holderTokens(address holder) external view returns (uint256[] memory);
> ```
> 
> I believe this was the bug of your interface code and I have fixed it. Because your implementation contract has address argument on holderTokens method and that is inconsistent with your original interface. For your information, here is the link to your BondingShareV2 contract on mainnet. https://etherscan.io/address/0x2dA07859613C14F6f05c97eFE37B9B4F212b5eF5#readContract
> 
> Thank you very much.

I appreciate the attention to detail @kfujita0520. Great job!